packages/banner-thermometer/src/banner-thermometer.ts

```diff
@@ -72,16 +72,20 @@ export class DonationBannerThermometer
                 : nothing}
             </div>
           </div>
-          <div class="donate-goal">${this.currentGoalMessage}</div>
+          ${this.goalMessageMode !== 'off'
+            ? html` <div class="donate-goal">${this.currentGoalMessage}</div> `
+            : nothing}
         </div>
       </div>
     `;
   }
 
   private get thermometerValueTemplate(): TemplateResult {
-    return html`
-      <div class="thermometer-value">${this.currentAmountDisplayValue}</div>
-    `;
+    return this.currentAmountMode === 'off'
+      ? html`${nothing}`
```

Comment:
suggestion: take `html` wrap off of nothing as it does not need a template wrap.

Reply:
When I remove the `html`, it gives me an error that it's not a `TemplateResult`. I think `nothing` only works within an `html` block.